src/QirRuntime/test/QIR-static/qir-driver.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation.

```

Comment:
Nit: It would be better to updated the licensing comments in one PR across the repo that doesn't have any meaningful changes...

Reply:
Agreed.

Explanation: Since this PR was the second time when Chris had to remind me to remove "All rights reserved" (which was the result of copy-paste of the code in this directory), I decided to secure myself from repeating the mistaken copy-paste pattern and removed all the "All rights reserved" from all the files where I likely to copy it again from during subsequent code changes.